pkg/plugins/policies/core/rules/subsetutils/subset.go

```diff
@@ -0,0 +1,339 @@
+package subsetutils
```

Comment:
In this file are just copies of functions we had in `rules.go`, you can skip reviewing this